packages/authentication/types/user.ts

```diff
@@ -5,19 +5,31 @@ export interface User {
   newEmail: string
   isNewEmailConfirmed: boolean
   referralCode: string
-  avatar: {
-    fullSize: string
-    small: string
-  }
-  firstName: string
```

Comment:
There seem to be some inconsistencies about the usage of `firstName` and `lastName` vs. `name`. I do not think that `firstName` and `lastName` are used on the frontend, so I deleted these variables in favor of `profile.name`.